src/client/app/components/UIOptionsComponent.tsx

```diff
@@ -5,39 +5,39 @@
 import * as React from 'react';
 import sliderWithoutTooltips, { createSliderWithTooltip } from 'rc-slider';
 import * as moment from 'moment';
-import { Button, ButtonGroup } from 'reactstrap';
+import { Button, ButtonGroup, Dropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap';
 import { TimeInterval } from '../../../common/TimeInterval';
 import TooltipHelpComponent from './TooltipHelpComponent';
 import ExportContainer from '../containers/ExportContainer';
 import ChartSelectContainer from '../containers/ChartSelectContainer';
 import ChartDataSelectContainer from '../containers/ChartDataSelectContainer';
-import { ChangeBarStackingAction } from '../types/redux/graph';
+import { ChangeBarStackingAction, ChangeSortingOrderAction } from '../types/redux/graph';
 import ChartLinkContainer from '../containers/ChartLinkContainer';
 import { ChartTypes } from '../types/redux/graph';
 import 'rc-slider/assets/index.css';
+import { ComparePeriod, SortingOrder } from '../utils/calculateCompare';
```

Comment:
For consistency, it would be great if this import came before the CSS import. Not a super important fix.